lib/src/testing_library/react/render/render.dart

```diff
@@ -0,0 +1,83 @@
+/// https://testing-library.com/docs/react-testing-library/api#render
+@JS()
+library over_react_test.src.testing_library.react.render.render;
+
+import 'dart:html' show Element;
+
+import 'package:js/js.dart';
+import 'package:over_react/over_react.dart' show ReactElement, UiFactory;
+import 'package:react/react_client/js_backed_map.dart' show JsBackedMap;
+import 'package:test/test.dart' show addTearDown;
+
+import 'package:over_react_test/src/testing_library/dom/queries/types.dart';
+import 'package:over_react_test/src/testing_library/react/render/types.dart';
+
+/// The element that [render] places the UI within.
+///
+/// Stored / used to allow queries to be used without having to specify a container, similar
+/// to how `screen` works in the RTL JS library.
+///
+/// Automatically torn down / null'd in the `tearDown` of any test that calls [render].
+///
+/// __FOR INTERNAL USE ONLY.__ Never export outside this package.
+Element get currentRtlRenderContainer => _currentRtlRenderContainer;
+Element _currentRtlRenderContainer;
```

Comment:
This was necessary to get the "top level" queries to work without forcing the consumer to specify the render container.

Reply:
I definitely set this up before I figured out how to get `screen` working... so maybe the top level queries can just require a `container` like the JS API does?